src/main/scala/mesosphere/marathon/api/akkahttp/AkkaHttpMarathonService.scala

```diff
@@ -73,7 +73,7 @@ class AkkaHttpMarathonService(
       async {
         val oldHandler = await(oldHandlerF)
         logger.info(s"Shutting down Akka HTTP service on ${config.httpPort()}")
-        val unbound = await(oldHandler.unbind())
+        val _ = await(oldHandler.unbind())
```

Comment:
Do you think we can just write `await(oldHandler.unbind())`?

Reply:
Sadly, no. This will eat up the exceptions in the `await` block.